ci/test-docker-in-docker.sh

```diff
@@ -18,17 +18,18 @@ if [[ "${IMAGE}" ]]; then
     export "CROSS_TARGET_${TARGET_UPPER//-/_}_IMAGE"="${IMAGE}"
 fi
 
-source=$(dirname "${BASH_SOURCE[0]}")
-source=$(realpath "${source}")
-home=$(dirname "${source}")
+ci_dir=$(dirname "${BASH_SOURCE[0]}")
+ci_dir=$(realpath "${ci_dir}")
+. "${ci_dir}"/shared.sh
+project_home=$(dirname "${ci_dir}")
```

Comment:
We can probably export this in `shared.sh` and then also use it for the `mkcargotemp`, as something like `export PROJECT_HOME=$(dirname "${ci_dir}")`.